charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -40,6 +40,18 @@ spec:
       resources:
         requests:
           storage: {{ .Values.cassandra.cassandraLibDirVolume.size | default "5Gi" }}
+    {{- if and (eq .Values.medusa.storage "local") ( .Values.medusa.podStorage ) }}
+    additionalVolumes:
+      - mountPath: /mnt/backups
```

Comment:
I installed from your branch with these properties:

```yaml
cassandra:
  heap:
    size: 768M
    newGenSize: 256M
  resources:
    requests:
      cpu: 1
      memory: 1600Mi
    limits:
      cpu: 1
      memory: 1600Mi
  datacenters:
  - name: dc2
    size: 3
medusa:
  enabled: true
  storage: local
  podStorage:
    size: 10Gi
    storageClass: standard
```

The `medusa` container fails with:

```
[2021-03-24 13:37:12,656] INFO: Init service
DEBUG:root:Loading storage_provider: local
[2021-03-24 13:37:12,656] DEBUG: Loading storage_provider: local
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/cassandra/medusa/service/grpc/server.py", line 158, in <module>
    medusa_pb2_grpc.add_MedusaServicer_to_server(MedusaService(config), server)
  File "/home/cassandra/medusa/service/grpc/server.py", line 45, in __init__
    self.storage = Storage(config=self.config.storage)
  File "/home/cassandra/medusa/storage/__init__.py", line 72, in __init__
    self.storage_driver = self._connect_storage()
  File "/home/cassandra/medusa/storage/__init__.py", line 96, in _connect_storage
    return LocalStorage(self._config)
  File "/home/cassandra/medusa/storage/abstract_storage.py", line 39, in __init__
    self.driver = self.connect_storage()
  File "/home/cassandra/medusa/storage/local_storage.py", line 27, in connect_storage
    driver = LocalStorageDriver(key=self.config.base_path)
  File "/home/cassandra/.local/lib/python3.6/site-packages/libcloud/storage/drivers/local.py", line 128, in __init__
    raise LibcloudError('The base path is not a directory')
libcloud.common.types.LibcloudError: <LibcloudError in None 'The base path is not a directory'>
``` 
You need to add the volume mount in the `medusa` container as well as in the `medusa-restore` container.